smack-tcp/src/main/java/org/jivesoftware/smack/tcp/XmppTcpTransportModule.java

```diff
@@ -580,6 +582,21 @@ protected XmppTcpNioTransport(ModularXmppClientToServerConnectionInternal connec
             super(connectionInternal);
         }
 
+        @Override
+        public StreamOpenAndCloseFactory getStreamOpenAndCloseFactory() {
+            return new StreamOpenAndCloseFactory() {
+                @Override
+                public StreamOpen createStreamOpen(CharSequence to, CharSequence from, String id, String lang) {
+                    StreamOpen streamOpen = new StreamOpen(to, from, id, connectionInternal.connection.getConfiguration().getXmlLang(), StreamOpen.StreamContentNamespace.client);
```

Comment:
That line is pretty long, and I don't like having method calls to fetch arguments. Please introduce a variable for xmlLang and use it. This also improves readability of the code.